### PR TITLE
Autotrain functionality

### PR DESCRIPTION
**Pull request by davidberenstein1957**
# Autotrain Functionality

- some minor spelling changes `HuggingFace`-> `Hugging Face`
- added support for datasnapshot with 5 samples when selecting text in various places
- direct integration with `autotrain` to be able to export and train models without any code

### CODE REVIEW

File: my_app/pages/autotrain-project.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+hf_auth_token, api = hf_login_flow()
+
+user_info = api.whoami()
+namespaces = [user_info["name"]] + [org["name"] for org in user_info["orgs"]]
+
+projects = get_projects(hf_auth_token)
+project_ids = [proj["proj_name"] for proj in projects]
+
+target_namespace = st.selectbox(
+    "Hugging Face namespace",
```
Comment (dvsrepo) [inline my_app/pages/autotrain-project.py:40]: I imagine is difficult but can we try to align this with HF terminology, would this be either the Organization or User namespace? Maybe just adding a brief comment explaining what we mean by namespace. I might just have missed this terminology from HF docs, if so please ignore!

File: my_app/pages/autotrain-project.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+)
+dataset_argilla_name = dataset_argilla.split("/")[-1]
+dataset_argilla_workspace = dataset_argilla.split("/")[0]
+get_data_snapshot(
+    dataset_argilla_name, dataset_argilla_workspace, query="status: Validated"
+)
+
+input_model = st.text_input(
+    "Input Model [from the hub](https://huggingface.co/models)",
+    value="bert-base-uncased",
```
Comment (dvsrepo) [inline my_app/pages/autotrain-project.py:60]: is there a problem if we provide a more recent model as default, like: https://huggingface.co/microsoft/deberta-v3-base or we'll we break AutoTrain :-D

**Review by dvsrepo — CHANGES_REQUESTED** [review]
This looks awesome! Just a few suggestion and a more general comment: I've seen in some places you refer to demos, would it be possible to refer to them as `apps` or something along the lines, or it would be too much? Something in between demos and apps?

File: my_app/pages/autotrain-project.py (hunk 2 above)
Comment (davidberenstein1957) [reply my_app/pages/autotrain-project.py:60]: Yes, this should work better.

File: my_app/pages/autotrain-project.py (hunk 1 above)
Comment (dvsrepo) [reply my_app/pages/autotrain-project.py]: 👍

**Review by dvsrepo — APPROVED** [review]

**Pull request merged**